generated/nirfmxwlan/nirfmxwlan.proto

```diff
@@ -31,27 +590,3421 @@ enum Boolean {
+  MEASUREMENT_TYPES_UNSPECIFIED = 0;
+  MEASUREMENT_TYPES_TXP = 1;
+  MEASUREMENT_TYPES_POWERRAMP = 2;
+  MEASUREMENT_TYPES_DSSSMODACC = 4;
+  MEASUREMENT_TYPES_OFDMMODACC = 8;
+  MEASUREMENT_TYPES_SEM = 16;
+}
+
+enum MechanicalAttenuationAuto {
+  option allow_alias = true;
+  MECHANICAL_ATTENUATION_AUTO_UNSPECIFIED = 0;
+  MECHANICAL_ATTENUATION_AUTO_FALSE = 0;
+  MECHANICAL_ATTENUATION_AUTO_TRUE = 1;
+}
+
+enum OfdmGuardIntervalType {
+  option allow_alias = true;
+  OFDM_GUARD_INTERVAL_TYPE_UNSPECIFIED = 0;
+  OFDM_GUARD_INTERVAL_TYPE_1_4 = 0;
+  OFDM_GUARD_INTERVAL_TYPE_1_8 = 1;
+  OFDM_GUARD_INTERVAL_TYPE_1_16 = 2;
+}
+
+enum OfdmLtfSize {
+  option allow_alias = true;
+  OFDM_LTF_SIZE_UNSPECIFIED = 0;
+  OFDM_LTF_SIZE_NOT_APPLICABLE = -1;
+  OFDM_LTF_SIZE_4_X = 0;
+  OFDM_LTF_SIZE_2_X = 1;
+  OFDM_LTF_SIZE_1_X = 2;
+}
+
+enum OfdmModAccAcquisitionLengthMode {
+  option allow_alias = true;
+  OFDM_MOD_ACC_ACQUISITION_LENGTH_MODE_UNSPECIFIED = 0;
+  OFDM_MOD_ACC_ACQUISITION_LENGTH_MODE_OFDMMODACC_ACQUISITION_LENGTH_MODE_MANUAL = 0;
+  OFDM_MOD_ACC_ACQUISITION_LENGTH_MODE_OFDMMODACC_ACQUISITION_LENGTH_MODE_AUTO = 1;
+}
+
+enum OfdmModAccAmplitudeTrackingEnabled {
+  option allow_alias = true;
+  OFDM_MOD_ACC_AMPLITUDE_TRACKING_ENABLED_UNSPECIFIED = 0;
+  OFDM_MOD_ACC_AMPLITUDE_TRACKING_ENABLED_FALSE = 0;
+  OFDM_MOD_ACC_AMPLITUDE_TRACKING_ENABLED_TRUE = 1;
+}
+
+enum OfdmModAccAveragingEnabled {
+  option allow_alias = true;
+  OFDM_MOD_ACC_AVERAGING_ENABLED_UNSPECIFIED = 0;
+  OFDM_MOD_ACC_AVERAGING_ENABLED_FALSE = 0;
+  OFDM_MOD_ACC_AVERAGING_ENABLED_TRUE = 1;
+}
+
+enum OfdmModAccCalibrationDataValid {
+  option allow_alias = true;
+  OFDM_MOD_ACC_CALIBRATION_DATA_VALID_UNSPECIFIED = 0;
+  OFDM_MOD_ACC_CALIBRATION_DATA_VALID_FALSE = 0;
+  OFDM_MOD_ACC_CALIBRATION_DATA_VALID_TRUE = 1;
+}
+
+enum OfdmModAccChannelEstimationType {
+  option allow_alias = true;
+  OFDM_MOD_ACC_CHANNEL_ESTIMATION_TYPE_UNSPECIFIED = 0;
+  OFDM_MOD_ACC_CHANNEL_ESTIMATION_TYPE_OFDMMODACC_CHANNEL_ESTIMATION_TYPE_REFERENCE = 0;
+  OFDM_MOD_ACC_CHANNEL_ESTIMATION_TYPE_OFDMMODACC_CHANNEL_ESTIMATION_TYPE_REFERENCE_AND_DATA = 1;
+}
+
+enum OfdmModAccCommonClockSourceEnabled {
+  option allow_alias = true;
+  OFDM_MOD_ACC_COMMON_CLOCK_SOURCE_ENABLED_UNSPECIFIED = 0;
+  OFDM_MOD_ACC_COMMON_CLOCK_SOURCE_ENABLED_FALSE = 0;
+  OFDM_MOD_ACC_COMMON_CLOCK_SOURCE_ENABLED_TRUE = 1;
+}
+
+enum OfdmModAccEvmUnit {
+  option allow_alias = true;
+  OFDM_MOD_ACC_EVM_UNIT_UNSPECIFIED = 0;
+  OFDM_MOD_ACC_EVM_UNIT_OFDMMODACC_EVM_UNIT_PERCENTAGE = 0;
+  OFDM_MOD_ACC_EVM_UNIT_OFDMMODACC_EVM_UNIT_DB = 1;
+}
+
+enum OfdmModAccFrequencyErrorEstimationMethod {
+  option allow_alias = true;
+  OFDM_MOD_ACC_FREQUENCY_ERROR_ESTIMATION_METHOD_UNSPECIFIED = 0;
+  OFDM_MOD_ACC_FREQUENCY_ERROR_ESTIMATION_METHOD_OFDMMODACC_FREQUENCY_ERROR_ESTIMATION_METHOD_DISABLED = 0;
+  OFDM_MOD_ACC_FREQUENCY_ERROR_ESTIMATION_METHOD_OFDMMODACC_FREQUENCY_ERROR_ESTIMATION_METHOD_INITIAL_PREAMBLE = 1;
```

Comment:
Note: not part of this change, but this is a good example of what we're up against with these acronyms.

`OFDM_MOD_ACC_FREQUENCY_ERROR_ESTIMATION_METHOD_OFDMMODACC_FREQUENCY_ERROR_ESTIMATION` Would be a redundant-or-annoying enum prefix. We need to strip off the prefix in enums.py, because a redundant one will be added by the grpc-device codegen. Those can be addressed in the API review story, although it's more efficient to do it before writing examples and tests. That logic is in scrapigen.


`OFDMMODACC_` vs `OFDM_MOD_ACC_` is something that we have to keep an eye on and maybe make changes for. The .NET API has good enum names (i.e.,  `OfdmModAccFrequencyErrorEstimationMethod`) but they chose to tokenize some of those acronyms differently from the enum values in the C API (i.e., `RFMXWLAN_ATTR_OFDMMODACC_RESULTS_POWER_PPDU_PEAK_POWER_MAXIMUM`). The `grpc-device` enum value prefixer uses the enum-name (`OfdmModAccFrequencyErrorEstimationMethod`) as the input and converts it to UPPER_SNAKE, which introduces the extra `_`s.